sample.html

```diff
@@ -19,8 +17,7 @@
     "playerBrandingId": "26e2e3c1049c4e70ae08a242638b5c40",
     'debug':true,
     "skin": {
-      "config": "",
```

Comment:
Why are we removing this?

Reply:
Do we not need the iframe example?

Reply:
@aldigun said to remove it. I'm assuming it's because it is outside the scope of a simple example.